src/main/java/seedu/address/model/Entity/Mentor.java

```diff
@@ -8,7 +8,6 @@
 public class Mentor extends Entity {
     private Phone phone;
     private Email email;
-    private List<Team> teams;
     private Name organization;
```

Comment:
So are we naming the variable for mentor's name as organization? Or does it mean something else?

Reply:
Oh this name is just the organization name, but the mentor's actual name attribute is part of the parent Entity class.

Reply:
Id and name will be supered to the parent constructor for all entities

Reply:
Oh right I see I see